SofaKernel/modules/SofaBaseTopology/SparseGridTopology.cpp

```diff
@@ -1419,8 +1426,8 @@ void SparseGridTopology::buildVirtualFinerLevels()
     _virtualFinerLevels[0]->_fillWeighted.setValue( _fillWeighted.getValue() );
     _virtualFinerLevels[0]->init();
 
-    sout<<"SparseGridTopology "<<getName()<<" buildVirtualFinerLevels : ";
-    sout<<"("<<newnx<<"x"<<newny<<"x"<<newnz<<") -> "<< _virtualFinerLevels[0]->getNbHexahedra() <<" elements , ";
+    msg_info()<<"SparseGridTopology "<<getName()<<" buildVirtualFinerLevels : " << msgendl;
+    msg_info()<<"("<<newnx<<"x"<<newny<<"x"<<newnz<<") -> "<< _virtualFinerLevels[0]->getNbHexahedra() <<" elements , " << msgendl;
```

Comment:
It is better (more efficient, more clear) to have a single message with multiple lines in it instead of two message.